Source/Editor/DualityEditor/DualityEditorApp.cs

```diff
@@ -847,18 +850,44 @@ public static void ReadPluginSourceCodeContentData(out string rootNamespace, out
 				}
 			}
 		}
-		public static void InitPluginSourceCode()
+
+		/// <summary>
+		/// Initializes the solution unless there already is an sln file.
+		/// </summary>
+		/// <returns>The path to the solution file</returns>
+		public static string InitPluginSourceCode()
```

Comment:
After changing the solution file path property, the return value of this method is no longer used and it shouldn't really be used to retrieve information anyway - consider changing the return type to `void`.